contents/blog/why-we-killed-our-ai-product-assistant.mdx

```diff
@@ -0,0 +1,103 @@
+
+## The assistant becomes the architect 
+Lots of beta users loved Max, but a comparison that kept coming up internally was Clippy; a paperclip animated assistant from Microsoft Office in the late 90’s, known for its intrusive, and unhelpful pop-ups (It was widely disliked and was removed in Office 2007).
+
+![automation screenshot](https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/0822_SM_Clippy_help_o6nlh3_49954d3c4c.webp)
+<Caption>Clippy was ahead of its time. Prove me wrong.</Caption>
+
+Clippy was the Kool-Aid Man busting through your wall; Max AI was opt-in and useful, but still it's instructive. Assistants don’t work when they’re a novelty UI. Whether or not users think it’s a novelty UI is something you have to take ownership of, especially if you're building an AI-native product. 
+
+Deciding how to address this was a hot topic at PostHog. A single Slack thread blew up with over 90 comments! The discussion was pretty intense, hitting on all the key angles from engineering to marketing. Did the mascot add to the user experience, or did it doom perception of PostHog AI as [‘just a chatbot'](/blog/ai-community-answers).
+
+![automation screenshot](https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/Screenshot_2025_11_10_at_3_05_26_PM_b886c5699e.png)
+
+In the end, we decided to axe Max. Not because we hate joy (we like it so much we [risked tanking our conversion rate](/blog/why-os)). We did it because the mascot was a limiting factor.
+
+Max AI became PostHog AI. Not just a character that lives in one corner, but the underlying architecture of PostHog product intelligence (sorry for sneaking a ‘not just’ statement in there).
+
+![automation screenshot](https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/Operation_Axe_Max_dcde574e08.png)
+<Caption>Don’t worry, we asked Max if he was ok with it.</Caption>
+
+### Why our AI got noticeably better since beta
+Long before this pivot, and as it was happening, the real work of improving PostHog AI was happening deep within PostHog. The <SmallTeam slug="posthog-ai"/> was building a shared infrastructure with the same philosophy as [HogQL](/docs/sql): a single system for each small team to extend.
+
+**Without it:** 
+- fragmented experience
+- duplicated effort
+- maintenance nightmare
+- Teams stuck building simple features from scratch instead of advanced stuff by default
+
+**With it:** 
+- shared agent system any product can extend
+- reusable components that work everywhere
+- consistent UX patterns
+- platform-level improvements that benefit everyone automatically
+
+Overhauling the guts was a good move. It enables deterministic tool‑calling, tighter context packing, and mode‑specific reasoning. 
+
+Translation: fewer “sorry I can’t do that yet” moments, faster replies, cheaper inference. 
+
+Key differences from older architectures:
+- **No hallucination**: The agent checks read_taxonomy before assuming event names exist
```

Comment:
Let's not say `no hallucinations` it might very well hallucinate, let's say `less hallucinations` or better: `Grounding`.

Rather than saying `read_taxonomy` which is an internal, let's say: `The agent always checks your data before giving an answer`